src/seedu/addressbook/AddressBook.java

```diff
@@ -486,14 +501,35 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            /*if (!Collections.disjoint(wordsInName, keywords)) {
                 matchedPersons.add(person);
+            }*/
+            if(keyMatchName(keywords, wordsInName)){
+            	matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
-
     /**
+     * Matches names ignoring case sensitivity
+     * 
+     * @param keywords for searching
+     * @param wordsInName 
+     * @return
+     */
+    private static boolean keyMatchName(Collection<String> keywords, Set<String> wordsInName) {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.